central/scannerdefinitions/handler/handler.go

```diff
@@ -51,6 +52,7 @@ const (
 	v4StorageDomain   = "https://storage.googleapis.com/scanner-v4-test"
 	mappingFile       = "redhat-repository-mappings/mapping.zip"
 	cvssFile          = "nvd-bundle/nvd-data.tar.gz"
+	v4VulnFile        = "output.json.zst"
```

Comment:
perhaps for another PR, but `output.json.zst` isn't very descriptive. Can we update this to `vulnerabilities.json.zst` or something?

Reply:
https://issues.redhat.com/browse/ROX-21942